src/app.js

```diff
@@ -149,13 +149,16 @@ export default async logger => {
       }
     )
     fastify.post(
-      `/:org(${SLUG_REGEXP_STR})/${SERVICE_1_1}`,
+      `/:org/${SERVICE_1_1}`,
       {
         errorHandler: soapErrorHandler,
         prefixTrailingSlash: 'both',
         schema: SoapRequestSchema
       },
       async (request, reply) => {
+        if (request.params && request.params.org) {
+          request.params.org = helpers.orgId(request.params.org)
```

Comment:
We should never override the original request parameters. This makes it hard to debug. Just add a line below `sites =` to normalize the org parameters, then use org everywhere.

```
  const sites =
    site &&
    typeof site === 'object' &&
    typeof site.string === 'string' &&
    site.string > ''
      ? [site.string]
      : site && typeof site === 'object' && Array.isArray(site.string)
      ? site.string
      : []
const org = helpers.org(request.params.org)
```

Also, rename the helper to simply `org` since it's no longer an orgId.